application/views/admin/token/emailwarning.php

```diff
@@ -72,6 +77,16 @@
         $('#tokenSendCounter').text(window.countdownTimerTokenSend);
     },1000);
 
+    $('#sendTokenInvitationsNow').on('click', function(evt){
+        clearInterval(intervaltoRenew);
```

Comment:
Didn't test, but i don't like $.css, you can not remive a class + add a class ?

For example if you have a class 'animate-red' addClass('animate-red'); with css

```
.animate-red{color:red;}
.animate-red{animation-duration:1s;}
```

when you set animate-red class : the animation happen.

Reply:
> Didn't test, but i don't like $.css, you can not remove a class + add a class ?

It was already being handled like this before.
I wouldn't update it right now, at least not in this bug.
I know it can be enhanced, I am not sure right now is the best time.